eccehomo/utils.py

```diff
@@ -1,16 +1,17 @@
-from urlparse import urlparse
+import six
+
+if six.PY2:
+    from urlparse import urlparse
+else:
+    from urllib.parse import urlparse
```

Comment:
Don't you prefer `from six.moves.urllib.parse import urlparse` https://pythonhosted.org/six/#module-six.moves ?![bitmoji](https://render.bitstrips.com/v2/cpanel/7136782-187400382_14-s1-v1.png?transparent=1&palette=1&width=246)